db/migrate/20210225190611_add_metrics_disk_read_write_latency.rb

```diff
@@ -0,0 +1,52 @@
+class AddMetricsDiskReadWriteLatency < ActiveRecord::Migration[6.0]
+  include MigrationHelper
+
+  DISK_LATENCY_METRIC_COLUMNS = %i[
+    disk_devicereadlatency_absolute_average
+    disk_kernelreadlatency_absolute_average
+    disk_totalreadlatency_absolute_average
+    disk_queuereadlatency_absolute_average
+    disk_devicewritelatency_absolute_average
+    disk_kernelwritelatency_absolute_average
+    disk_totalwritelatency_absolute_average
+    disk_queuewritelatency_absolute_average
+  ].freeze
+
+  def up
+    drop_view("metrics")
+    drop_view("metric_rollups")
+
+    add_disk_latency_metric_columns("metrics_base")
+    add_disk_latency_metric_columns("metric_rollups_base")
+
+    create_view("metrics")
+    create_view("metric_rollups")
+  end
+
+  def down
+    drop_view("metrics")
+    drop_view("metric_rollups")
```

Comment:
wonder if this would be cleaner as `recreate_metrics_views`...something like...

```ruby
recreate_metrics_views do
  remove_disk_latency_metric_columns("metrics_base")
  remove_disk_latency_metric_columns("metric_rollups_base")
end
```

and then under the covers recreate is just

```
# In migration_helper.rb
def recreate_metrics_views
  drop_view("metrics")
  drop_view("metric_rollups")

  yield

  create_metrics_view("metrics")
  create_metrics_view("metric_rollups")
end
```